fetch_issues.py

```diff
@@ -30,8 +30,18 @@ def fetch_total_results():
     global url, response, total_results
     url = f'{jira_server}/sr/jira.issueviews:searchrequest-xml/temp/SearchRequest.xml?jqlQuery={encoded_query}&tempMax=1&pager/start=1'
     response = requests.get(url)
+
+    if (response.status_code == 400):
+        print(f'Error fetching data from Jira, status code: {response.status_code}')
+        print(response.text)
+        exit(1)
     result = objectify.fromstring(response.text)
-    return int(result.channel.issue.attrib['total'])
+
+    count = int(result.channel.issue.attrib['total'])
+    if count == 0:
+        print('No results found for the given JQL query.')
```

Comment:
seemed easier to exit with an error here  as the rest of the code throughout doesn't really handle 0 issues very well as no xml file will be generated and what if no xml file is for a different reason...